account_payment_order/models/account_move_line.py

```diff
@@ -12,6 +12,9 @@ class AccountMoveLine(models.Model):
     partner_bank_id = fields.Many2one(
         comodel_name="res.partner.bank",
         string="Partner Bank Account",
+        compute="_compute_partner_bank_id",
```

Comment:
I think making this a computed field could maybe lead to unexpected results. Would you consider, instead of updating the move line's partner_bank_id directly to change the logic here: https://github.com/OCA/bank-payment/blob/13.0/account_payment_order/models/account_move_line.py#L58 and fetch either the move line's partner_bank_id when it's set, or the move's invoice_partner_bank_id, or else any of the partner's banks?

Reply:
That solution is the first solution he presents, but he wants to solve the reason why he did not transfer the bank account, therefore it was not well received. Now the third solution is presented, you can read the solution conversation where this solution is recommended.

Reply:
Oh sh*t that was your first proposal but @pedrobaeza did not like it. Well, then I will refrain from reviewing this PR further.

Reply:
Haha no problem 👍